pkg/apis/awsprovider/v1alpha1/awsclusterproviderconfig_types.go

```diff
@@ -35,6 +35,12 @@ type AWSClusterProviderConfig struct {
 
 	// SSHKeyName is the name of the ssh key to attach to the bastion host.
 	SSHKeyName string `json:"sshKeyName,omitempty"`
+
+	// CACertificate is a PEM encoded CA Certificate for the control plane nodes.
+	CACertificate []byte `json:"caCertificate,omitempty"`
```

Comment:
What's the reason for moving this to config from status?

Reply:
Total hack for #348

Reply:
That, but it does allow for a user to override the CA material for a cluster...  In the future we may want to extend this to a type that could be either defined inline or a reference to an external source of truth (k8s secret, aws ssm, etc)

Reply:
In addition to the other pints, it also allows to retain the certificates on pivot, which wasn't happening before.